src/Native/System.Native/pal_networking.cpp

```diff
@@ -218,11 +209,7 @@ extern "C" int32_t GetHostEntryForName(const uint8_t* address, HostEntry* entry)
         return ConvertGetAddrInfoAndGetNameInfoErrorsToPal(result);
     }
 
-    *entry = {
-        .CanonicalName = nullptr,
-        .AddressListHandle = reinterpret_cast<void*>(info),
-        .IPAddressCount = 0
-    };
+    *entry = {.CanonicalName = nullptr, .AddressListHandle = reinterpret_cast<void*>(info), .IPAddressCount = 0};
```

Comment:
Same comment as earlier... I realize this is being done by a tool following a rule, so I'll stop commenting on it... I'm just questioning the rule.